pacmacs.el

```diff
@@ -130,18 +131,58 @@
         :column column
         :type 'wall))
 
-(defun pacmacs--make-pill (row column anim-name points)
+(defun pacmacs--make-pill (row column anim-name points type)
```

Comment:
This function doesn't feel like `pacmacs--make-pill` anymore, because it is more general and can make any `type` of object. Maybe something like `pacmacs-make-object` will be more appropriate?

Also you could then express `pacmacs--make-wall-cell` in terms of `pacmacs--make-object _ _ _ nil 'wall`.

What do you think?